src/app/checklist/[checklist_slug]/[locale]/page.tsx

```diff
@@ -0,0 +1,138 @@
+            // TODO: Extract checkitems and add them to JSON-LD
+            step: [],
+        }),
+    };
+}
+
+const ChecklistPage = ({
+    params: { checklist_slug: slug, locale },
+}: {
+    params: ChecklistParam;
+}) => {
+    const checklist = ChecklistService.getInstance().getChecklistData(
+        slug,
+        locale
+    );
+    const {
+        frontmatter: {
+            title,
+            description,
+            author_username,
+            author_name,
+            tags = [],
+        },
+        availableLocales,
+    } = checklist;
+
+    return (
+        <>
+            <Script
+                id="ld-json"
+                type="application/ld+json"
+                dangerouslySetInnerHTML={generateJSONLD(checklist)}
+            />
+            <div className="flex flex-col container mx-auto">
+                <div className="flex justify-between">
+                    <div className="flex flex-col gap-4 my-10">
+                        <div className="flex justify-between">
+                            {title && (
+                                <h2 className="text-4xl font-mono">{title}</h2>
+                            )}
+                        </div>
+                        <div className="checklist-author">
+                            <Link
+                                href={
+                                    author_username
+                                        ? `https://github.com/${author_username}/`
+                                        : "#"
+                                }
+                                target="_blank"
+                                className="flex gap-2 text-blue items-center"
+                            >
+                                {author_username && (
+                                    <Avatar
+                                        username={author_username}
+                                        size={35}
+                                    />
+                                )}
+                                {author_name && <h5>{author_name}</h5>}
+                            </Link>
+                        </div>
+                        <h5 className="text-light-gray">{description}</h5>
+                        <Tags tags={tags} />
+                        <div className="flex text-blue gap-2 items-center">
+                            <CircleStackIcon title="Storage" className="h-4" />
+                            <h5>Checks are saved to your local storage</h5>
+                        </div>
+                    </div>
+                    <div className="mt-10">
+                        <LocalesDropdown
+                            availableLocales={availableLocales}
+                            hrefPrefix={`/checklist/${slug}/`}
+                        />
+                    </div>
+                </div>
+
+                <article className="prose prose-invert prose-ul:list-none max-w-3xl px-10 py-4 bg-dark-gray rounded-md">
```

Comment:
Did you test responsiveness?

Reply:
Not on this page, will do

Reply:
It's "usable", but very ugly
Will fix, specially the home page

Reply:
It's not perfect, but definitely better now
Please take a look: https://devchecklists-rangelfinal.vercel.app/